cmd/notify.go

```diff
@@ -0,0 +1,100 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/elgs/gojq"
+	"github.com/pkg/errors"
+	"io/ioutil"
+	"log"
+	"strings"
+	"time"
+)
+
+var (
+	timeLayout              = time.RFC1123
+	lastUpdateCheckFilePath = makeCephNanoPath("last_update_check")
+)
+
+func checkUpdateNotification() {
+	if !shouldCheckURLVersion(lastUpdateCheckFilePath) {
+		return
+	}
+	latestVersion, latestVersionURL, err := getLatestVersionAndURL()
+	if err != nil {
+		log.Fatal(err)
+		return
+	}
+	localVersion := getLocalVersion()
+	if latestVersion != localVersion {
+		writeTimeToFile(lastUpdateCheckFilePath, time.Now().UTC())
+		fmt.Printf("\nThere is a newer version of ceph nano available (%s).\nDownload it here:%s\n\n",
+			latestVersion, latestVersionURL)
+	}
+}
+
+func getLatestVersionAndURL() (string, string, error) {
```

Comment:
Please call https://github.com/ceph/cn/blob/bdf9e3a8b78649699df1be07a06889704959d875/cmd/update-check.go#L48-L49 instead of writing the same function.